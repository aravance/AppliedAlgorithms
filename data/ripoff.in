10 4 5
100 50 -20 60 30
-10 -30 -50 20 70
9 3 4
150 100 -200
-100 -300 -100
-200 100 150
20 3 16
10 10 -10 -20 10
100 100 100 100 100
100 100 100 100 100
100 100 100 100 100
19 4 15
10 10 -10 -20 10
100 100 100 100 100
100 100 100 100 100
100 100 100 100
151 6 26
5000 -10 -10 -10 -10 -10
5000 -10 -10 -10 -20 -10
5000 -10 -10 -10 -10 -10
5000 -10 -10 -10 -20 -10
5000 -10 -10 -10 -30 -10
5000 -10 -10 -10 -20 -10
5000 -10 -10 -20 -10 -10
5000 -10 -10 -30 -10 -10
5000 -10 -10 -20 -10 -10
5000 -10 -10 -20 -10 -10
5000 -10 -10 -20 -10 -10
5000 -10 -10 -20 -10 -10
5000 -10 -10 -10 -10 -10
5000 -10 -20 -10 -70 -80
5000 -10 -20 -10 -70 -80
5000 -10 -20 -10 -70 -80
5000 -10 -20 -10 -70 -80
5000 -10 -20 -10 -70 -80
5000 -10 -10 -10 -10 -10
5000 -10 -10 -10 -10 -10
5000 -10 -10 -10 -10 -10
5000 -10 -10 -10 -10 -10
5000 -20 -10 -20 -20 -20
5000 -10 -30 -40 -30 -40
5000 -10 -20 -10 -70 -80
5000
24 3 15
9999 9999 9999 9999 9999
100 100 -300 100 100 -300
100 100 -300 100 100 -300
100 100
9999 9999 9999 9999 9999
14 5 3
10 10 10 10 -10
10 10 10 10 -10
10 10 10 10
12 10 13
1 1000 1 1000
1 1000 1 1000
1 1000 1 1000
200 10 30
5262 -254 -5467 -9163 -5418 724 9806 6985 3131 -7032
4738 -3990 -2821 -3842 -4201 -5325 8651 5655 -4017 2933
-4602 884 -2745 -5963 5526 3886 6726 -7519 3073 5758
-7328 -6471 -5022 -7072 -3084 -1217 1712 1720 8133 4625
-7904 4532 241 1445 -1137 4380 5432 2803 -2355 -2618
-399 -7557 -7916 -9424 -6692 6166 7026 -31 819 7175
170 -2533 2634 -1247 7955 8924 9816 2616 2179 3881
199 5851 -7021 9053 -5283 -458 -7757 8576 9574 3483
1122 2799 2713 -3177 -1150 -9906 7628 -705 -4823 -9685
-4586 -2089 -7171 1930 4858 -6087 -2324 7371 5725 -7020
249 -7196 4478 1378 6676 6652 -8387 -6091 3983 -7509
2751 5988 -5567 1 5290 9306 604 1966 -8148 7554
-2807 -8689 -2649 -446 -3825 2401 -5717 -1475 -4310 2382
2092 -1791 -3770 -3392 -8657 6064 8147 -2312 4356 8550
-1255 -2761 -8615 723 -9164 3923 3839 -5753 -6663 4390
-4233 -5094 -4249 4093 -3231 -3062 -1345 4538 -3594 -6278
2400 -1000 9628 -2972 7545 7320 -822 4288 2050 8126
6566 -3738 2492 -7111 -1422 7583 -4301 -5440 5209 4231
-8634 7339 -1773 -6010 7250 -9434 -395 2737 -1465 2832
-8708 -4529 2232 -7097 -1074 -2157 1338 4728 -3894 9462
200 10 23
5262 -254 -5467 -9163 -5418 724 9806 6985 3131 -7032
4738 -3990 -2821 -3842 -4201 -5325 8651 5655 -4017 2933
-4602 884 -2745 -5963 5526 3886 6726 -7519 3073 5758
-7328 -6471 -5022 -7072 -3084 -1217 1712 1720 8133 4625
-7904 4532 241 1445 -1137 4380 5432 2803 -2355 -2618
-399 -7557 -7916 -9424 -6692 6166 7026 -31 819 7175
170 -2533 2634 -1247 7955 8924 9816 2616 2179 3881
199 5851 -7021 9053 -5283 -458 -7757 8576 9574 3483
1122 2799 2713 -3177 -1150 -9906 7628 -705 -4823 -9685
-4586 -2089 -7171 1930 4858 -6087 -2324 7371 5725 -7020
249 -7196 4478 1378 6676 6652 -8387 -6091 3983 -7509
2751 5988 -5567 1 5290 9306 604 1966 -8148 7554
-2807 -8689 -2649 -446 -3825 2401 -5717 -1475 -4310 2382
2092 -1791 -3770 -3392 -8657 6064 8147 -2312 4356 8550
-1255 -2761 -8615 723 -9164 3923 3839 -5753 -6663 4390
-4233 -5094 -4249 4093 -3231 -3062 -1345 4538 -3594 -6278
2400 -1000 9628 -2972 7545 7320 -822 4288 2050 8126
6566 -3738 2492 -7111 -1422 7583 -4301 -5440 5209 4231
-8634 7339 -1773 -6010 7250 -9434 -395 2737 -1465 2832
-8708 -4529 2232 -7097 -1074 -2157 1338 4728 -3894 9462
200 5 46
5262 -254 -5467 -9163 -5418 724 9806 6985 3131 -7032
4738 -3990 -2821 -3842 -4201 -5325 8651 5655 -4017 2933
-4602 884 -2745 -5963 5526 3886 6726 -7519 3073 5758
-7328 -6471 -5022 -7072 -3084 -1217 1712 1720 8133 4625
-7904 4532 241 1445 -1137 4380 5432 2803 -2355 -2618
-399 -7557 -7916 -9424 -6692 6166 7026 -31 819 7175
170 -2533 2634 -1247 7955 8924 9816 2616 2179 3881
199 5851 -7021 9053 -5283 -458 -7757 8576 9574 3483
1122 2799 2713 -3177 -1150 -9906 7628 -705 -4823 -9685
-4586 -2089 -7171 1930 4858 -6087 -2324 7371 5725 -7020
249 -7196 4478 1378 6676 6652 -8387 -6091 3983 -7509
2751 5988 -5567 1 5290 9306 604 1966 -8148 7554
-2807 -8689 -2649 -446 -3825 2401 -5717 -1475 -4310 2382
2092 -1791 -3770 -3392 -8657 6064 8147 -2312 4356 8550
-1255 -2761 -8615 723 -9164 3923 3839 -5753 -6663 4390
-4233 -5094 -4249 4093 -3231 -3062 -1345 4538 -3594 -6278
2400 -1000 9628 -2972 7545 7320 -822 4288 2050 8126
6566 -3738 2492 -7111 -1422 7583 -4301 -5440 5209 4231
-8634 7339 -1773 -6010 7250 -9434 -395 2737 -1465 2832
-8708 -4529 2232 -7097 -1074 -2157 1338 4728 -3894 9462
0
