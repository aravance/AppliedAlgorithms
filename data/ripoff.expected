220
-100
1390
1310
-250
98790
-20
6006
213821
144745
207839
